Loop/View Controllers/SettingsTableViewController.swift

```diff
@@ -69,6 +69,10 @@ final class SettingsTableViewController: UITableViewController, DailyValueSchedu
         super.viewDidDisappear(animated)
 
         dataManager.rileyLinkManager.setDeviceScanningEnabled(false)
+        // TODO: Do not upload every time we exit settings.  Perhaps have a check for change?
```

Comment:
This check exists:

https://github.com/LoopKit/Loop/pull/682/files#diff-5b9067796f682df0e6723dfbb8aa0e6bR320